app.js

```diff
@@ -61,22 +61,12 @@ wss.on('connection', function connection (ws) {
 
   cmd = 'ssh';
   args = dir ? [host, '-t', 'cd \'' + dir.replace(/\'/g, "'\\''") + '\' ; exec ${SHELL} -l'] : [host];
-  cwd = process.env.HOME;
-  env = {};
 
-  pty_spawn_env = {
+  term = pty.spawn(cmd, args, {
     name: 'xterm-256color',
     cols: 80,
-    rows: 30,
-    env: env
-  };
-
-  // omit cwd if ! exist, as in case of first login flow using pam_mkhomedir
-  if(fs.existsSync(cwd)){
-    pty_spawn_env.cwd = cwd;
-  }
```

Comment:
I take it this doesn't work ?

Reply:
It does work. It is just unnecessary. The idea of 1ed707559870e0b4e1db16556ac1057b7fc3bc5b was to revert the change of checking if cwd existed before setting cwd on pty_spawn_env. The point here is that we can just never set cwd since it doesn't matter what $PWD is before you execute ssh.